4 - Tables and Persistence/7 - Saving Data/4.7 Lab/EmojiDictionary/EmojiDictionary/EmojiTableViewController.swift

```diff
@@ -0,0 +1,110 @@
+
+import UIKit
+
+class EmojiTableViewController: UITableViewController {
+    
+    var emojis = [Emoji(symbol: "😀", name: "Grinning Face", detailDescription: "A typical smiley face.", usage: "happiness"),
```

Comment:
You will need to change things up so that emojis are only set to this list if there is nothing saved in the past!